url, copy right, author had changed.

gurezo opened the pull request:
### Proposed Changes


### has taken the following confirmation

 + [x] checked command succeeds with `gulp build`
 + [x] has not dropped coverage

gurezo committed:
url, copy right, author had changed.

sizuhiko commented on bower.json:
```diff
@@ -2,15 +2,15 @@
   "description": "WebGPIO and WebI2C API polyfill",
   "authors": [
-    "club-wot"
+    "chirimen-oh"
```
ここは複数記述できるので、併記しても良いかなーと思います

```json
    "club-wot",
    "chirimen-oh"
```

gurezo replied:
そうですね、良い案だと思います。
@MSakamaki さんからのお返事も聞いてから判断しましょう。

MSakamaki replied:
個人的にはどちらでも！、一応club-wot入れておきますか。

gurezo replied:
修正しますね

gurezo committed:
- authors had added.
- name, description had changed.
- keyword had added.
- copy right had changed.
